app/models/story.rb

```diff
@@ -569,11 +569,14 @@ def log_moderation
       m.moderator_user_id = self.editor.try(:id)
     end
     m.story_id = self.id
-
     if all_changes["is_expired"] && self.is_expired?
       m.action = "deleted story"
     elsif all_changes["is_expired"] && !self.is_expired?
       m.action = "undeleted story"
+    elsif check_for_description_repost(all_changes)
+      comment = self.comments.find(&:new_record?)
```

Comment:
Please move comment saving up to the controller. It's bad enough that `Story` creates `Moderation` records, let's not make it more of a god object.

Reply:
Done.